djangocms_admin_style/templates/admin/inc/branding.html

```diff
@@ -2,4 +2,5 @@
 <div id="header-logo">
     <img src="/static/djangocms_admin_style/img/django-cms-logo.png" alt="django CMS">
 </div>
-<h1 id="site-name">{{ site_header|default:_('example.com') }}</h1>
\ No newline at end of file
+<h1 id="site-name">{{ site_header|default:_('example.com') }}</h1>
+<div id="header-btn" class="btn"><a href="/admin"><input class="icon-arrow-right" type="button" value="Back to example.com"></a></div>
```

Comment:
the link leads to admin and not to the website.
also example.com should be `{{ site_header|default:_('example.com') }}`
also why button inside a link?